src/scripts/libraryMenu.js

```diff
@@ -516,10 +516,9 @@ function createToolsMenuList(pluginItems) {
         icon: 'bug_report'
     });
     links.push({
-        name: globalize.translate('TabNotifications'),
+        name: globalize.translate('Notifications'),
```

Comment:
Changed the translation key from `TabNotifications` to `Notifications` because the latter is required for the plugin catalog categories.